apps/condo/public/tls/windows/russian_trusted_root_ca.cer

```diff
@@ -0,0 +1,33 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Should it be at public repo?!

Reply:
This file can be downloaded from https://www.sberbank.com/ru/certificates/ios page.
I don't see any problems to host it ourselves.
This way we will not depend on external sources.

Reply:
Extracted changes, related to ADR#16 into different PR, that will be opened later with task DOMA-7045